src/components/adminArea/Drawer/UserEditForm.tsx

```diff
@@ -21,11 +23,32 @@ const UserEditForm = ({ currentUser, setIsLoading }) => {
     const [role, setUserRole] = useState(currentUser?.role || Roles.Regular);
     const [badgesList] = useAtom(atomBadgesList);
     const [userId] = useAtom(currentUserId);
+    const [options, setOptions] = useState<NameSpace[]>([]);
+    const [selectedNamespaces, setSelectedNamespaces] = useState<NameSpace[]>([]);
 
     const handleChangeBadges = (_event, newValue: Badge[]) => {
         setBadges(newValue);
     };
 
+    const handleChangeNameSpaces = (_event, newValue: NameSpace[]) => {
+        setSelectedNamespaces(newValue);
+    };
+    useEffect(() => {
+        const fetchNamespaces = async () => {
+            try {
+                const namespaces = await NameSpacesApi.getNameSpaces();
```

Comment:
issue: Right now, if NameSpacesApi.getNameSpaces() fails, the catch {} will silently swallow the error and you’ll have no idea why your UI might be stuck empty